### PR TITLE
Add platform-monitor container to one image.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**

### CODE REVIEW

File: rules/docker-platform-monitor.mk [hunk 1]
```diff
@@ -6,3 +6,8 @@ $(DOCKER_PLATFORM_MONITOR)_DEPENDS += $(SONIC_CONFIG_ENGINE)
 $(DOCKER_PLATFORM_MONITOR)_LOAD_DOCKERS = $(DOCKER_BASE)
 
 SONIC_DOCKER_IMAGES += $(DOCKER_PLATFORM_MONITOR)
+SONIC_INSTALL_DOCKER_IMAGES += $(DOCKER_PLATFORM_MONITOR)
+
+$(DOCKER_PLATFORM_MONITOR)_CONTAINER_NAME = platform-monitor
+$(DOCKER_PLATFORM_MONITOR)_RUN_OPT += --net=host --privileged -t
+$(DOCKER_PLATFORM_MONITOR)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
```
Comment (lguohan) [inline rules/docker-platform-monitor.mk]: is this needed? @taoyl-ms, can you check?

Comment (oleksandrivantsiv) [inline rules/docker-platform-monitor.mk]: Currently it is not needed. I've added it to be similar to other containers.

Comment (oleksandrivantsiv) [inline rules/docker-platform-monitor.mk]: are you ok with container name? Maybe it should be something sorter (e.g. pm)?

Comment (lguohan) [inline rules/docker-platform-monitor.mk]: how about pmon

Comment (oleksandrivantsiv) [inline rules/docker-platform-monitor.mk]: Will change it

**Pull request merged**